packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Build an API service integration
+excerpt: Learn how to build and register an API service integration to the Okta Integration Network.
+layout: Guides
+---
+
+<ApiLifecycle access="ea" />
+
+This guide explains how to build and submit an API service integration to the Okta Integration Network (OIN) for review and publishing.
+
+---
+
+**Learning outcomes**
+
+* Learn how to build an API service integration that can be published in the OIN.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+   > **Note:** Your org needs to have the API Service Integration feature enabled. Contact your Okta account team to enable the feature.
+* A service app that needs to access Okta APIs for your customer
+
+
+---
+
+## Overview
+
+A service-to-service app where where a backend service or a daemon calls [Okta management APIs](/docs/reference/core-okta-api/) for a tenant (Okta org) can be published in the Okta Integration Network (OIN) as an API service integration.
+API service integrations access Okta APIs using the OAuth 2.0 Client Credentials grant flow, where access is not associated with a user and resources can be restricted with scoped access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
+
+> **Note:** Currently, Okta only supports OAuth APIs listed in [Scopes and supported endpoints](/docs/guides/implement-oauth-for-okta/main/#scopes-and-supported-endpoints). We're working towards supporting scopes for all Okta API endpoints.
+
+If you have an API service integration for your app in the OIN, your customers can configure your integration to implement the OAuth 2.0 Client Credentials flow with their Okta tenant org. Each customer Okta org has its own authorization server. When a customer authorizes your API service integration to access their org, Okta generates a unique set of credentials (client ID and client secret) for that org.
```

Comment:
I might be reading it incorrectly but 

> to implement the OAuth 2.0 Client Credentials flow with their Okta tenant org

makes it sound like the customer will implement? This detail (multi-tenancy concept) gets missed, so I think it's good to call it out, but do you think it's covered in your section below on Collect and save customer credentials"?

Reply:
Replace "implement" with "use". I think it's good to have this description in the overview. The Collect and save customer credentials section is part of "building the integration" part, so it's spelling out what to save.